.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
If you are using the sum.js to prove that js is working (or any local files to test setup of a new package), I'd temporarily add them to your gitignore to avoid committing them (as they are unrelated to this specific project you're working on!